api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPTCompletionCohortQueries.java

```diff
@@ -154,24 +154,6 @@ public CohortDefinition getTxCurrWithTPTCompletion() {
                 hivMetadata.getRestartConcept().getConceptId()),
             mapping));
 
-    compositionCohortDefinition.addSearch(
-        "B1",
```

Comment:
This is being used in the composition, any reason to remove it?

Reply:
Yes! For requirement 8 "Patients who already completed INH" it is asked to delete the whole B1 query hence I also removed it from the compositions where it is applied.

Reply:
B1 had two parts: B1part1(The one to delete) and B1part2, For TPTEligiblePatientListCohortQueries, yet for Completion which uses the same query, only called for B1. and this was giving an error when running the module.